meta-client/src/main/java/org/apache/helix/metaclient/factories/MetaClientConfig.java

```diff
@@ -19,23 +19,32 @@
  * under the License.
  */
 
-class MetaClientConfig {
+import org.apache.helix.metaclient.constants.MetaClientConstants;
+
+public class MetaClientConfig {
 
   public enum StoreType {
     ZOOKEEPER, ETCD, CUSTOMIZED
```

Comment:
Interesting to see "etcd" here. Should we mention this here yet or just say customized? (could there be licensing issue? )

Reply:
If i understand currectly, etcd should not have any license issue. It is ok to mention it here but we do not plan to have any implementation in near future.